packages/flutter/lib/src/material/icon_button.dart

```diff
@@ -141,7 +141,7 @@ class IconButton extends StatelessWidget {
   /// or an [ImageIcon].
   const IconButton({
     Key? key,
-    this.iconSize = 24.0,
+    this.iconSize = 24,
```

Comment:
Since we're not systematically removing all of the `.0`s from this file, best to make no changes at all so that this file doesn't show up in the changed-files list.

Reply:
This was an inaccuracy of my reversion. Adding the `.0` back.